pkg/config/utils.go

```diff
@@ -247,6 +248,83 @@ func checkConfig() error {
 	return nil
 }
 
+// processBasePath adjusts the current working dir by making it relative to the provided base path
+// NOTE: this function is not in use, we leave it here (in utils) in case we need it in the future
+func processBasePath() error {
```

Comment:
I will leave it up to you, but I suggest that if you are going to keep this around, you make it a bit more generic, and have it take arguments. Maybe call it `makeRelativePath` and take `basePath` and `targetPath` as arguments?